views/demo.pug

```diff
@@ -0,0 +1,29 @@
+extends layout.pug
+
+block head
+	.navbar
```

Comment:
This block needs relooked at for proper component design. All HTML elements should have a class as we never style elements directly.